src/index.ts

```diff
@@ -33,14 +33,20 @@ app.get('/status', async (_req, res) => {
   const status = await bot.status();
   jsonExt(status, res);
 });
-app.get('/earnings/:rotationId', async (req, res) => {
-  const rotationId = req.params.rotationId;
-  const data = await bot.storage.earningsFile(Number(rotationId)).get();
+app.get('/bids', async (_req, res) => {
+  const currentFrameId = await bot.currentFrameId();
+  const nextFrameId = currentFrameId + 1;
```

Comment:
Is this right? Shouldn't active bids go against the current frame id?

Reply:
It is, but the bidsFile used tomorrow's cohortId/frameId (what we're now calling frameIdAtCohortActivation) to retrieve today's bidding. I should probably rename nextFrameId to frameIdAtCohortActivation in this code.